src/issuemd-core.js

```diff
@@ -1,43 +1,43 @@
-'use strict';
+(function (root, factory) {
 
-! function () {
+    'use strict';
```

Comment:
what is purpose of moving `'use strict';` inside function? Should it be moved inside functions in every module? Is there something wrong with having it at top of file? Is it for after concatenation so that it is only applied to same functions it was intended for?